app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Select from "react-select";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import Col from "react-bootstrap/Col";
+import LinkExternal from "../LinkExternal";
+
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import {
+  getConfig_nimbusConfig,
+  getConfig_nimbusConfig_channels,
+} from "../../types/getConfig";
+
+// TODO: find this doco URL
+const AUDIENCE_DOC_URL =
+  "https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FJT&title=Project+Nimbus";
+
+type FormAudienceConfig = Pick<
+  getConfig_nimbusConfig,
+  "channels" | "firefoxMinVersion" | "targetingConfigSlug"
+>;
+
+export const FormAudience = ({
+  experiment,
+  config,
+}: {
+  experiment: getExperiment_experimentBySlug;
+  config: FormAudienceConfig;
+}) => {
+  const {
+    channels,
+    firefoxMinVersion,
+    targetingConfigSlug,
+    populationPercent,
+    totalEnrolledClients,
+    proposedEnrollment,
+    proposedDuration,
+  } = experiment;
+
+  const channelsOptions = config.channels?.filter(
+    (item): item is getConfig_nimbusConfig_channels => item !== null,
+  );
+  const channelsDefaultValue = channels?.map((channel) =>
+    channelsOptions?.find((option) => option?.value === channel),
+  );
+
+  return (
+    <Form data-testid="FormAudience">
+      <Form.Group>
+        <Form.Row>
+          <Form.Group as={Col} controlId="channel" md={8} lg={8}>
+            <Form.Label>Channel</Form.Label>
+            <Select
+              isMulti
+              name="channel"
+              data-testid="channel"
+              defaultValue={channelsDefaultValue}
+              options={channelsOptions}
+            />
+          </Form.Group>
+          <Form.Group as={Col} controlId="minVersion">
+            <Form.Label>Min Version</Form.Label>
+            <Form.Control
+              as="select"
+              data-testid="minVersion"
+              defaultValue={firefoxMinVersion || ""}
+            >
+              <SelectOptions options={config.firefoxMinVersion} />
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} controlId="targeting">
+            <Form.Label>Advanced Targeting</Form.Label>
+            <Form.Control
+              as="select"
+              data-testid="targeting"
+              defaultValue={targetingConfigSlug || ""}
+            >
+              <SelectOptions options={config.targetingConfigSlug} />
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+      </Form.Group>
+
+      <Form.Group className="bg-light p-3">
```

Comment:
Nit: I think `p-4` is a bit closer to Figma, as well as removing `className="mb-3"` on `InputGroup` on L99 and L126.